udata/core/dataset/apiv2.py

```diff
@@ -0,0 +1,152 @@
+import logging
+
+from flask import url_for
+
+from udata.api import api, apiv2, API, fields
+
+from .api_fields import (
+    badge_fields,
+    DEFAULT_MASK,
+    org_ref_fields,
+    resource_fields,
+    spatial_coverage_fields,
+    temporal_coverage_fields,
+    user_ref_fields,
+)
+from .models import (
+    UPDATE_FREQUENCIES, DEFAULT_FREQUENCY, DEFAULT_LICENSE
+)
+from .permissions import DatasetEditPermission
+
+DEFAULT_PAGE_SIZE = 50
+
+log = logging.getLogger(__name__)
+
+ns = apiv2.namespace('datasets', 'Dataset related operations')
+resources_parser = api.parser()
```

Comment:
Woops, a better suggestion this time:

```suggestion
resources_parser = apiv2.parser()
```